### ISSUE TITLE
dataloader error

### ISSUE THREAD

**Issue by cy-yong** [voxelmorph/voxelmorph#449]
![image](https://user-images.githubusercontent.com/79499756/176125895-e8ee707e-d4f9-4528-9081-1a2d5a3c7475.png)
when i try to load my file from a txt file, it remain me that it is not a file.

**Comment by adalca** [comment]
you probably need to provide the full path, did you? @ahoopes might have good intuitions

**Comment by ahoopes** [comment]
Yes, probably the case. What are your command line arguments (and path to training images)?
